change_log.md

```diff
@@ -0,0 +1,2271 @@
+# Change Log
+
+## Version 1.2.2 - Revised to add New Modes
+The Part 1, Part 2, & Part 3 Schemas include   corrections and enhancements since the issue of the Version 1.2 CEN Specification documents for NeTEx. 

```

Comment:
```suggestion
The Part 1, Part 2, & Part 3 schemas include corrections and enhancements since the release of the version 1.2 CEN specification documents for NeTEx. 

```